### DISCUSSION TITLE
UI updates via WSS not working in simple app (deployed on Render)

### DISCUSSION THREAD

**Discussion by mafshin** [zauberzeug/nicegui#1898] [Q&A]
### Question

Hi all.

Having the simple demo app deployed to Render, I see that control update messages does not work properly via websockets.
It works locally but on Render I see no response for UI WSS messages.

https://github.com/mafshin/nicegui-demo
https://nicegui-demo.onrender.com/

Do you have any idea of what's missing or incorrectly configured?

**Comment by rodja** [comment]
Should work. See https://github.com/zauberzeug/nicegui/discussions/1455. @Smug246 how far is your tutorial?

**Comment by mafshin** [comment]
Just deployed the same application using [Docker](https://github.com/mafshin/nicegui-demo/blob/main/Dockerfile) and WS is working fine on Render. 

https://github.com/mafshin/nicegui-demo/blob/main/Dockerfile 

Docker version: https://nicegui-demo.onrender.com
Python version: https://nicegui-demo-python.onrender.com

**Reply by rodja** [reply]
Would you like to create a short tutorial on https://github.com/zauberzeug/nicegui/wiki?

**Reply by mafshin** [reply]
I will do as soon as I find a free time slot
